src/state/README.md

```diff
@@ -52,3 +52,103 @@ app.container.on('connected', async () => {
     await UserData.delete('my-value');
 });
 ```
+### IndexedDB
+Any work with indexedDB or the userData plugin as a whole should be done inside the 'connected' callback method
+
+``` javascript
+import { UserData } from 'springroll';
+
+app.container.on('connected', async () => {
+  // connect to your database
+  let response = await userData.OpenDb('dbName');
```

Comment:
See my comment in the `UserData` class. If we don't actually need the constructor then these calls should be `UserData` not `userData`. If we actually do end up needing the constructor, make sure to include that call above here, and comment that it is explicitly required for the IDB stuff.